src/region/RegionAnnotations.tsx

```diff
@@ -142,10 +142,11 @@ export default class RegionAnnotations extends React.Component<Props, State> {
                 )}
 
                 {/* Layer 3b: New, unsaved annotation description popup, if 3a is ready */}
-                {isCreating && staged && canReply && targetRef && (
+                {isCreating && staged && targetRef && (
```

Comment:
I think this changes the user experience to where the popup is always shown, even if the user is still drawing, which isn't desirable. It should only be shown once the mouse button is released.